requirements.txt

```diff
@@ -41,3 +42,7 @@ tomd==0.1.3
 wagtail==2.9
 wagtail-cache==1.0.0
 whitenoise==5.1.0
+
+# django-file-form==2.1.3
+# Fork that supports storing temporary files on S3:
+-e git+https://github.com/balazs-endresz/django-file-form.git@custom-storage#egg=django_file_form
```

Comment:
Good chance of getting the changes included in the main project you think?

Reply:
I think so. Especially since they've tried to do something like this before. Maybe the problem was just that they didn't use the proper django File/storage api (which is a bit weird). But I guess we'd have a better case to make if we tried it out on heroku (dev) first.

Btw, django-file-form itself has some tests, which did pass locally, except for the selenium based ones. (I suspect it's still not easy to set up selenium locally. And last time I tried to get travis to run a whole test suite locally I gave up after a day. I haven't tried that with circle-ci yet, which is used on django-file-form.) 

So, django-file-form probably needs some more tests and documentation changes. I've added only one more config option, so updating the docs shouldn't take too long. I guess it'd be helpful to add tests for a custom (local) storage backend too. Or see how they mock it for boto and do something similar.

If getting a new version released takes too long then in the meanwhile you can fork it under OTF and install it the same way as I did here. Not ideal but works fine in the short term.